Vizzuality/heco-invest
# feat: Devise uses deliver_later method

This PR contains two changes related to Devise :).

First of all, I am moving all locales to `zu.yml`, because these locales are pushed to transifex and we want to have these strings at transifex.

Second change is that devise mails are now send via `deliver_later` method and they use correct `locale` of user --> text of emails are based on account language or `ui_language` variable of appropriate record.

## Testing instructions

All tweaked devise mails share same logic so It should be enough to test one. I think that good candidate is for example forgotten password at backoffice. Create Admin with Spanish ui_language and use forgotten password feature --> such admin should receive email in Spanish.

## Tracking

https://vizzuality.atlassian.net/browse/LET-815
https://vizzuality.atlassian.net/browse/LET-816
